### PR TITLE
Entries: Note that the normalizer can return any value

### PR DESCRIPTION
**Pull request by Arkni**
Ref jquery-validation/jquery-validation#2054

### CODE REVIEW

File: entries/normalizer.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 		<ul>
 			<li><p>Doesn't change the elements' value, it only changes the value used for validation.</p></li>
 			<li><p>Gets the value passed as argument, and "this" within it references the corresponding <code>DOMElement</code>.</p></li>
-			<li><p>Needs to return a String value, otherwise it will throw a <code>TypeError</code> exception.</p></li>
+			<li><p>Can return any value including <code>null</code> and <code>undefined</code>.</p></li>
```
Comment (staabm) [inline entries/normalizer.xml]: We should note thenversion since when this is supported

Comment (Arkni) [reply entries/normalizer.xml]: In that case, I will wait for the next release to update it.

Comment (staabm) [reply entries/normalizer.xml]: Next will be a minor non-feature release

Comment (Arkni) [reply entries/normalizer.xml]: Updated per suggestion

**Comment by Arkni** [comment]
Hi @staabm,

I totally forgot about this PR. Could you please merge it and if it's not too much, can you also deploy this change?

Thanks a lot :)

**Comment by staabm** [comment]
ohh missed it :-). merged it, will deploy when time allows.

**Comment by Arkni** [comment]
Thanks :)

**Pull request merged**